cli/cmd/install.go

```diff
@@ -820,17 +826,18 @@ func render(w io.Writer, values *l5dcharts.Values) error {
 
 	if values.Stage == "" || values.Stage == configStage {
 		for _, template := range templatesConfigStage {
-			files = append(files, &chartutil.BufferedFile{
-				Name: template,
-			})
+			files = append(files,
+				&chartutil.BufferedFile{Name: template},
+			)
 		}
 	}
 
 	if values.Stage == "" || values.Stage == controlPlaneStage {
 		for _, template := range templatesControlPlaneStage {
-			files = append(files, &chartutil.BufferedFile{
-				Name: template,
-			})
+			files = append(files,
+				&chartutil.BufferedFile{Name: template},
+				&chartutil.BufferedFile{Name: "smi-metrics-config.yaml"},
+			)
```

Comment:
Probably want to append `smi-metrics-config.yaml` outside of the loop!

Reply:
good catch, thanks